src/game.cpp

```diff
@@ -2444,8 +2444,15 @@ void Game::updatePlayerControl(const CameraOrientation &cam)
 	}
 #endif
 
-	client->setPlayerControl(control);
 	LocalPlayer *player = client->getEnv().getLocalPlayer();
+
+	// autojump if set: simulate "jump" key
```

Comment:
This would register the player as always jumping even though the player only jumps at obstacles. Maybe this can be left out?